infra/taskcluster-hook-pipeline-start.json

```diff
@@ -51,7 +51,9 @@
         "retries": 5,
         "routes": [
             "notify.email.release-mgmt-analysis@mozilla.com.on-failed",
-            "notify.irc-channel.#bugbug.on-failed"
+            "notify.irc-channel.#bugbug.on-failed",
+            "project.relman.services.project.bugbug.production.bugbug_data.latest",
```

Comment:
This way the task triggered by the hook (the data pipeline start task) will be indexed here, instead we have to index the actual tasks that do these things.